docs/setup.md

```diff
@@ -2,7 +2,146 @@
 title: 'Setup'
 ---
 
-# GitHub Actions [linux/mac/windows] {: #github-actions}
+# Build your wheel locally
+
+Before starting to configure cibuildwheel, it's useful to try to build a wheel
+on your local machine. It's much easier to debug problems on your machine than
+inside a CI system!
+
+Start with a clean checkout of your project, inside a new clean virtual
+environment. Make sure your `pip` is up to date, and then invoke:
+
+```shell
+pip wheel -w wheelhouse .
```

Comment:
This really should be `pipx run build` or `pipx run build --wheel` if a valid SDist is not possible. But that gets into more details than we want here, I think?

Reply:
I think so. The reason for using `pip wheel` was to match the cibuildwheel mechanism. Also all users will have `pip`, some won't have `pipx`. But when we switch over to `build` as the default, we should update this.